.circleci/config.yml

```diff
@@ -66,6 +66,9 @@ workflows:
       - prep-build-test-nft:
           requires:
             - prep-deps
```

Comment:
I don't think we need all these changes to introduce an erc1155 specific job, if you change the test path to `test/e2e/nft/import-erc1155.spec.js` it will be run in the `test-e2e-*-nft` job.